test/rubygems/test_gem.rb

```diff
@@ -198,6 +198,7 @@ def assert_self_install_permissions
       'gems/foo-1/bin/foo.cmd' => prog_mode,
       'gems/foo-1/data/foo.txt' => data_mode,
     }
+    sleep 0.1 if win_platform?
```

Comment:
Can you add a comment here so years later on we can easily see why this was added?

Reply:
Done.  Also found errors on Travis, see below